### DISCUSSION TITLE
Fermata auto 1.8.0 re-install

### DISCUSSION THREAD

**Discussion by ccpoon1982** [AndreyPavlenko/Fermata#49] [Q&A]
Hi Andrew & all experts here, I accidentally click on update to fermata 1.8.2 while I'm using 1.8.0 with no problem at all with my toyota android auto. After updated to 1.8.2, my toyota can't show fermata auto nor fermata control at the apps launcher. Why? Any idea? 

How can I reinstall back to 1.8.0? At least with 1.8.0 there is no issue for me.
I have tried to look for 1.8.0 apk from website, and I have tried cleaned all apps in my mobile, re-installed it, but It doesn't works for me anymore. 

Thanks.
Poon

**Comment by AndreyPavlenko** [comment]
Hi,

What do you mean by "click on update"? Have you installed the google play version? For AA you need the github version. Please follow this instruction - https://forum.xda-developers.com/t/fermata-media-player-audio-and-video-player-for-android-auto.4079519/ .

**Reply by ccpoon1982** [reply]
Hi Andrey, thanks for replied. I meant this morning I went in this link https://github.com/AndreyPavlenko/Fermata/releases to click on 1.8.2 and download the latest version while I'm having 1.8.0 working well before hand. I'm totally regreted for my "itcy hand" this morning. 
Previously, I used AAAD to installed 1.8.0 & all works well. 
So now, what should I do? To make fermata auto appeared & come alive again in my android auto screen? Appreciated for your advice soon.
Thanks

**Reply by AndreyPavlenko** [reply]
All works well, because you used AAAD. Just use AAAD to install the new version. Manual apk installation does not work.

**Reply by ccpoon1982** [reply]
Thanks Andrey, let me try using AAAD again. Thanks for your feedback. Also in my previous version, 1.8.0 I faced one little issues. While driving, When I was showing some YouTube movie, once I make some right turn, left turn or reverse, my PVM will automatically display the camera view in the screen. Then when it switch back to the YouTube screen from PVM view after I have done the turning or reversing, then it will stucks in a blank black YouTube screen , the YouTube doesn't continue to play. This is the major reason why I was trying to click update from 1.8.0 to 1.8.2 yesterday. So, is this a common issue in Fermata Auto?
Thanks

**Reply by AndreyPavlenko** [reply]
This is not common, I don't have this issue. Try the latest version 1.8.3.

**Comment by Alejandro2454** [comment]
Estimado,
Extraño lo que usted dice. Con el ánimo de poder ayudarle podría usted
decir ¿que versión tiene de AA y que versión tiene del sistema android en
vuestro celular?
Saludos.
Alejandro

**Comment by Alejandro2454** [comment]
Para poder contribuir necesito información básica. ¿Cuál es tu versión de
Android Auto? ¿ Cuál es tu versión de Android de tu celular?
Saludos
Alejandro

**Reply by ccpoon1982** [reply]
Hi, my android auto version is 6.8.613354 release. My mobile is running on android version 10.
Also, My mobile does not "Root"

**Comment by Alejandro2454** [comment]
Prueba instalando AA  v6.9 o AA 7.0 beta, para descartar o no AA del
problema que tienes
https://www.apkmirror.com/apk/google-inc/android-auto/
saludos,
Alejandro

**Reply by ccpoon1982** [reply]
Hi Alexander, thanks. Just a question, for v6. 9 updates, do I still need to install fermata auto via github AAAD or I can use the Google play store version? Which one will works? 

Thanks

**Comment by Alejandro2454** [comment]
Hola, deberías usar github e instalar la versión de fermata que más te
acomode. Google Play store no.
Saludos
Alejandro
Pd: atención que en este momento en github hay una contradicción en lo que
dice sobre versión 1.8.3 dado que la mezcla con 1.8.2

**Reply by ccpoon1982** [reply]
Thanks Alexander. Let me try if there is any issue, I will post it here again, thanks for the advice.

**Comment by Alejandro2454** [comment]
No ha problema. Solo asegúrate de tener habilitado modo desarrollador y
fuentes desconocidas en AA

**Reply by ccpoon1982** [reply]
Hi Alexander there are 4 variants for v6. 9. So which one should I choose? What is the different among those 4 types?

Thanks

**Comment by Alejandro2454** [comment]
Es la arquitectura de la cpu. Instala de Google play la app del enlace. La
ejecutas y en system te aparece la arquitectura de la cpu. Mira donde dice
conjunto de instrucciones

**Comment by Alejandro2454** [comment]
Disculpa, va imagen de la app de google play

**Reply by ccpoon1982** [reply]
OK. Thank you. My phone is arm64_v8a
I found it
Thanks

**Comment by Alejandro2454** [comment]
hola mi amigo,
¿como te ha ido?
¿todo bien?
saludos
Alejandro
